server/main.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	secure := true
+	port := "8443"
+	directory := "/acm-cli"
+
+	// Verify certificate and key files for TLS
+	for _, crtFile := range []string{"tls.crt", "tls.key"} {
```

Comment:
Optional: Usually I prefer a CLI argument to indicate it should be HTTPS and then fail the container from starting if the cert isn't provided. The idea being that if it can't be served securely, then it shouldn't be served at all.